etc/profile-m-z/tutanota-desktop.profile

```diff
@@ -23,9 +27,17 @@ noblacklist ${HOME}/.mozilla
 whitelist ${HOME}/.mozilla/firefox/profiles.ini
 read-only ${HOME}/.mozilla/firefox/profiles.ini
 
+no3d
```

Comment:
So tutanota does not use hardware accelerated rendering?

Reply:
No, your PR adds it.

Well, Firefox does, GTK app do, chromium does, ... why should tutanota-desktop override it to use software rendering?

Reply:
We use a version control system called "git". Instead of saying that some old version ahd it, you can show that it had it and link to the commit which removed it.